src/parser/view_transform.c

```diff
@@ -13918,3 +13959,79 @@ mq_copy_sql_hint (PARSER_CONTEXT * parser, PT_NODE * dest_query, PT_NODE * src_q
 	}
     }
 }
+
+/*
+ * mq_update_sort_spec_expr() - update PT_VALUE located within the OVER clause of the analytic function.
+ *   return:
+ *   parser(in):
+ *   before_query(in): 
+ *   spec(in):
+ */
+static PT_NODE *
+mq_update_sort_spec_expr (PARSER_CONTEXT * parser, PT_NODE * select_list, PT_NODE * old_attrs, PT_NODE * new_attrs)
+{
+  PT_NODE *partition_by, *order_by;
+  PT_NODE *col, *link, *order_list, *order, *value;
+
+  for (col = select_list; col; col = col->next)
+    {
+      if (PT_IS_ANALYTIC_NODE (col))
+	{
+	  partition_by = col->info.function.analytic.partition_by;
+	  order_by = col->info.function.analytic.order_by;
+
+	  /* link partition and order lists together */
+	  for (link = partition_by; link && link->next; link = link->next)
+	    {
+	      ;			/* move link to the last node of partition_by list */
+	    }
+	  if (link)
+	    {
+	      order_list = partition_by;
+	      link->next = order_by;
+	    }
+	  else
+	    {
+	      order_list = order_by;
+	    }
+
+	  for (order = order_list; order; order = order->next)
+	    {
+	      PT_NODE *old_attr, *new_attr;
+	      int index;
+
+	      if (!PT_IS_VALUE_NODE (order->info.sort_spec.expr))
+		{
+		  continue;
+		}
+
+	      value = order->info.sort_spec.expr;
+	      /* retrieve the order-th node from the old_attrs */
+	      old_attr = pt_resolve_sort_spec_expr (parser, order, old_attrs);
+	      assert (old_attr != NULL);
```

Comment:
`assert (old_attr != NULL);` 만으로도 문제 없지만...
좀 더 안전하게 아래처럼 하는 것이 좋을 것 같습니다.
```suggestion
	      if (old_attr == NULL)
		{
		  assert (false);
		  return NULL;
		}
```